kzz7yh-a67606/cod-ve07v1_kzz7yh-a67606.xml

```diff
@@ -67,6 +67,7 @@
    </p>
             <div xml:id="kzz7yh-a67606-Dd1e115">
                <head xml:id="kzz7yh-a67606-Hd1e117">Quaestio 1</head>
+               <head type="question-title">Utrum quaelibet persona se et quanlibet aliam diligat spiritu sancto</head>
```

Comment:
I'm gonna normalize this to "quamlibet" since it is in the editorial header.
```suggestion
               <head type="question-title">Utrum quaelibet persona se et quamlibet aliam diligat spiritu sancto</head>
```